### PR TITLE
Simple script to talk to meetup API and receive JWToken on sign request

### PR DESCRIPTION
**Pull request by christiangubana**

### CODE REVIEW

File: test/live.js [hunk 1]
```diff
@@ -2,33 +2,49 @@
...
+console.log('Token - ' + token);
 
 Seneca({ legacy: false })
   .test()
   .use('promisify')
   .use('entity')
   .use('env', {
     // debug: true,
-    file: [__dirname + '/local-env.js;?'],
+    file: [__dirname + '/local-env.template.js'],
```
Comment (rjrodger) [inline test/live.js]: the template is just a template - copy to local-env.js

File: test/live.js [hunk 2]
```diff
@@ -2,33 +2,49 @@
...
     var: {
-      $MEETUP_KEY: String,
-      $MEETUP_NAME: String,
-      $MEETUP_CUSTID: String,
-      $MEETUP_ACCID: String,
+      $YOUR_CONSUMER_KEY: String,
```
Comment (rjrodger) [inline test/live.js]: not fully documented, but no need for the $ prefix here - it just indicates we want to hide the var value in logging

File: test/local-env-template.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 module.exports = {
```
Comment (rjrodger) [inline test/local-env-template.js]: revert to template, and copy this to local-env.js

**Review by rjrodger — CHANGES_REQUESTED** [review]

File: test/local-env-template.js (hunk 3 above)
Comment (christiangubana) [reply test/local-env-template.js]: Reverted all these changes to keep it clean and simple in live.js script.
The idea is to get the script generates JWToken when hitting the meetup API.

**Pull request closed without merging**